qiita_ware/util.py

```diff
@@ -117,104 +116,14 @@ def stats_from_df(df):
     out = {}
 
     for column in natsorted(df.columns):
+        if column in {'study_id'}:
```

Comment:
i mean... that is an easy change that can be made in the future with literally no impact anywhere else.